File: crates/deslop/tests/fixtures/fused-golden-php/rename_a.php
<?php

function route(int $weight, int $distance, string $carrier): string
{
    $score = $weight * 3 + $distance;
    if ($score > 900) {
        return $carrier . '-freight';
    }

    if ($score > 400) {
        return $carrier . '-ground';
    }

    return $carrier . '-parcel';
}

File: crates/deslop/tests/fixtures/fused-golden-rust/rename_a.rs
pub fn route(weight: i64, distance: i64, carrier: &str) -> String {
    let score = weight * 3 + distance;
    if score > 900 {
        return carrier.to_owned() + "-freight";
    }
    if score > 400 {
        return carrier.to_owned() + "-ground";
    }
    carrier.to_owned() + "-parcel"
}
